src/content/posts/aws-sysops-exam.md

```diff
@@ -0,0 +1,61 @@
+
+### Practice exams
+
+Practice exams are not just about scoring yourself to see if you'll pass. A good set of tests will provide a long explanation to each question and links to documentation where you can read more about it.
+
+1. [A Cloud Guru exam simulator](https://acloud.guru/exam-simulator). I had a mixed experience:
+   - **Solutions Architect Associate** exam questions are worth trying. Their difficulty is medium. I scored 70-80% on first attempts. So this did give me some feedback on areas to work on. I think its worth to attempt this a few times until you score 90%.
+   - **SysOps Administrator Associate** exam questions are way too easy and the database is quite small. As a result, I scored over 90% on my first attempt. Moreover, the way that ACG simulator works often results in answering a question which you might have already seen in previous session. As a result, you spend time on answering easy questions which you have seen before. Not very effective and not much to learn there.
+1. [Udemy, Jon Bonso - AWS Certified SysOps Administrator Associate Practice Exams](https://www.udemy.com/course/aws-certified-sysops-administrator-associate-practice-exams-soa-c01/). Because I felt under-prepared after ACG exams, I searched for other exam sources. Many people indicated that Jon Bonso's course is hard and has detailed explanations for each question. I can confirm that it is harder than the actual exam and you can learn a lot from the extra resources provided in each answer. There are 5 tests with unique questions. I have scored 69%, 78%, 87%, 92%, 76% few days before the actual exam where I scored 98%. If you agree with philosophy of over-preparing then definitely use this exam set.
+
+### Notes
+
+I've produced many pages of markdown notes from all resources that I used. I definitely recommend to **write down the critical things that you hear or read** while learning. On the day of my exam, I had 2 hours in train and in waiting room before the exam. I read all I had written before. It was very useful and I keep coming back to these notes today.
+
+## Final words
+
+I've never been a fan of certifications. I prefer hands-on experience with real-world problems to solve, bugs to debug and infrastructure to be built. Why I bothered with getting the certificate? First, I was looking to shift my career, from OpenStack environment to AWS. Getting certified gets the hiring process a bit easier for both sides. Certificate is a clear indication that I actually know the cloud that I am supposed to operate on. Second, I found the knowledge that I gained to be extremely useful on practical part of job interviews. In the end, I am glad I went through the process. If you are thinking about working on AWS, then going through the path I described above is probably a good choice.
```

Comment:
Shouldn't it be: Firstly,... Secondly,...  ?

Reply:
It's all very nice and helpful - also personally for me, because I want to repeat your journey :)

Additionally, you could elaborate some more on: 2 months of working with AWS. What use cases to realize?